tests/string/strncmp.c

```diff
@@ -0,0 +1,15 @@
+#include <string.h>
+#include <stdio.h>
+
+int main(int argc, char* argv[]) {
+    printf("%d\n", strncmp("a", "aa", 2));
+    printf("%d\n", strncmp("a", "aä", 2));
+    printf("%d\n", strncmp("\xFF", "\xFE", 2));
+    printf("%d\n", strncmp("", "\xFF", 1));
+    printf("%d\n", strncmp("a", "c", 1));
+    printf("%d\n", strncmp("a", "a", 2));
+
+    puts("test");
```

Comment:
puts hasn't been implemented yet.

Reply:
Ah, yeah I was using that to figure out why the tests weren’t working.  I must have forgotten to delete that line.